templates/etc/pam.d/sshd_el7.epp

```diff
@@ -0,0 +1,30 @@
+#%PAM-1.0
+auth	   required	    pam_sepermit.so
+auth       required     pam_faillock.so preauth<% if
```

Comment:
'preauth' should start its own line, ending with the end of the deny block (mid-line 5 atm)